src/core/tooltip/tooltip.js

```diff
@@ -0,0 +1,64 @@
+/* global window */
+
+import { createElement } from 'jsx-dom';
+import styles from './tooltip.css';
+
+function createTooltip(card, target) {
+  const wrapper = <hyper-gwent-tooltip style={{
+    display: 'none',
+    position: 'fixed',
+    transform: 'translate(10px, -60px)',
+    zIndex: 999999999,
```

Comment:
Yep, I went over the limit on this one 😄  The browser is normalizing to maximum value but a good old `10` could have been enough.